CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ cmake_minimum_required(VERSION 3.21)
 
 cmake_policy(SET CMP0025 NEW)
 
+# Loading save files created by a previous version of the game is
+# unsupported with one exception: the first three version components have to match
+# and the fourth version component has to be greater or equal.
```

Comment:
This description has no place in top level CMakeLists.txt. It should be moved near relevant implementation details, like in a comment before the function that checks save file version.

Reply:
Removing TWEAK made this comment redundant